src/components/SampleDistributionOLSEstimators/StandardNormalOLS.js

```diff
@@ -0,0 +1,90 @@
+/*eslint-disable no-unused-vars */
+import { useState, useEffect } from 'react';
+import Highcharts from 'highcharts';
+import HighchartsReact from 'highcharts-react-official'
+import BellCurve from 'highcharts/modules/histogram-bellcurve';
+import { distributionType, hypothesisTestingSampleArrayType } from '../../lib/types';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { dataFromDistribution } from '../../lib/stats-utils';
+import { sqrt } from 'mathjs';
+
+BellCurve(Highcharts);
+
+export default function StandardNormalOLS({ samples, sampleSize}) {
+  const [population] = useState(
+    dataFromDistribution('Normal', 2000, { mean: 0, standardDev: 1 })
```

Comment:
Why is a new population being generated here? We should be plotting the samples.